.github/workflows/get-values.yaml

```
@@ -0,0 +1,50 @@
+
+      - name: Update Devcontainer Hash
+        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'push' }}
+        id: update-hash
+        uses: ./.github/actions/update-devcontainer-hash
+        with:
+          branch: ${{ github.ref_name }}
+      - name: Get the PR number
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'merge_group' }}
+        id: find-pr-num
+        # Based on https://github.com/actions/checkout/issues/58#issuecomment-847922894
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:139: Double quote to prevent globbing and word splitting

(shellcheck)